config.h

```diff
@@ -1,9 +1,21 @@
-/* Some basic macro definitions for high performance.
- * If something is not supported by your system, turn it off.
- * This will be substituted by automatic script in future.*/
+/* Some basic macro definitions for high performance.git
```

Comment:
Damnit, no.  Sorry, bug with wayland + focus-follow-mouse is messing with me.  I'll update the PR momentarily.  Thanks for spotting this.